docs/content/Dialog2.mdx

```diff
@@ -177,3 +177,78 @@ render(<ShorthandExample2 />)
 | cancelButtonContent  | `React.ReactNode`                 | `"Cancel"` | The content to use for the cancel button.                                                                 |
 | confirmButtonContent | `React.ReactNode`                 | `"OK"`     | The content to use for the confirm button.                                                                |
 | confirmButtonType    | `"normal" │ "primary" │ "danger"` | `Button`   | The type of button to use for the confirm button.                                                         |
+
+## AlertDialog
+
+An `AlertDialog` is a special kind of dialog with more rigid behavior. It is used to alert or confirm a single user action. AlertDialogs can only have at most one button: no buttons or one to confirm the alert dialog. No custom rendering capabilities are provided for AlertDialog.
```

Comment:
Can we be more explicit here about when to use a ConfirmationDialog vs an AlertDialog?

Reply:
updated with a more explicit description, please let me know if that is a good direction. 👍🏽